### PR TITLE
:lipstick: Add new UI at modals

### PR DESCRIPTION
**Pull request by EvaMarco**
This PR solves this [task](https://tree.taiga.io/project/penpot/task/6036)

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -5,5 +5,6 @@
         "**/.lsp": true,
         "**/.shadow-cljs": true,
         "**/node_modules": true
-    }
+    },
```
Comment (superalex) [inline .vscode/settings.json]: Why is this included ?

Comment (EvaMarco) [reply .vscode/settings.json]: My bad, it should be on gitingnore and I didn't realize
